ballerina/Module.md

```diff
@@ -43,7 +268,7 @@ You need to set values for the following basic configuration parameters in the `
 |  password  |    The password of the DB server.    |
 |  database  | The name of the database to be used. |
 
-The following is a sample `Config.toml` file with the MySQL data store configuration. This is generated by the `bal persist generate` command.
+The following is a sample `Config.toml` file with the MySQL data store configuration. This will be generated by the `bal build` or `bal persist generate` command (depending on how you have configured the application).
```

Comment:
```suggestion
The following is a sample `Config.toml` file with the MySQL data store configuration. This will be generated by the `bal build` or `bal persist generate` command (depending on how you generate the client API).
```